uk.ac.kcl.inf.modelling.xdsml.henshin_xdsml_specification.ui.tests/.classpath

```diff
@@ -1,9 +1,7 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="src" path="src-gen"/>
-	<classpathentry kind="src" path="xtend-gen"/>

```

Comment:
This should be there. I know it means that on a fresh checkout you will get a build error, but the correct fix for that is to manually add the `xtend-gen/` folder to the project.